v21.1/deploy-cockroachdb-with-kubernetes-openshift.md

```diff
@@ -0,0 +1,329 @@
+	~~~
+
+	This lets you issue `oc` commands without having to specify the namespace each time.
+
+## Step 2. Install the Operator
+
+1. Navigate to your OpenShift web console and click on **OperatorHub**.
+
+1. Enter "cockroach" in the search filter. There are two tiles called **CockroachDB Operator**. Find the tile _without_ the `Marketplace` label (which requires a subscription).
+
+	<img src="{{ 'images/v21.1/cockroachdb-operator-openshift.png' | relative_url }}" alt="OpenShift OperatorHub" style="border:1px solid #eee;max-width:100%" />
+
+	Click the **CockroachDB Operator** tile and then **Install**.
+
+1. On the **Install Operator** page, select `cockroachdb` in the **Installed Namespace** dropdown and click **Install**.
+
+1. Validate that the Operator is running:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ oc get pods
+    ~~~
+
+	~~~
+	NAME                                  READY   STATUS    RESTARTS   AGE
+	cockroach-operator-65c4f6df45-h5r5n   1/1     Running   0          51s
+	~~~
+
+### Step 3. Start CockroachDB
+
+1. When the Operator is ready, click **View Operator** to navigate to the **Installed Operators** page.
+
+1. In the **CockroachDB Operator** tile, click **Create instance**.
+
+	<img src="{{ 'images/v21.1/cockroachdb-operator-instance-openshift.png' | relative_url }}" alt="OpenShift OperatorHub" style="border:1px solid #eee;max-width:100%" />
+
+1. Make sure **CockroachDB Version** is set to an Operator-supported version. For a list of compatible image names, see the [Operator deployment definition](https://github.com/cockroachdb/cockroach-operator/blob/master/manifests/operator.yaml) on GitHub.
+
+1. This will open the **Create CrdbCluster** page. By default, this deploys a 3-node secure cluster. Leave the other fields unchanged and click **Create**.
+
+1. Navigate to **Workloads** > **Pods** and observe the pods being created:
+
+	<img src="{{ 'images/v21.1/cockroachdb-operator-pods-openshift.png' | relative_url }}" alt="OpenShift OperatorHub" style="border:1px solid #eee;max-width:100%" />
+
+1. You can also use the command line to view the pods:
+
+	{% include copy-clipboard.html %}
+	~~~ shell
+	oc get pods
+	~~~
+
+	~~~
+	NAME                                  READY   STATUS    RESTARTS   AGE
+	cockroach-operator-65c4f6df45-h5r5n   1/1     Running   0          6m11s
+	crdb-tls-example-0                    1/1     Running   0          3m15s
+	crdb-tls-example-1                    1/1     Running   0          103s
+	crdb-tls-example-2                    1/1     Running   0          89s
+	~~~
+
+## Step 4. Create a secure client pod
+
+To use the CockroachDB SQL client, first launch a secure pod running the `cockroach` binary.
+
+This can be defined with the following YAML, which mounts the Operator's generated certificates:
+
+{{site.data.alerts.callout_success}}
+`spec.containers.image` should refer to the CockroachDB version you specified when [starting the nodes](#step-3-start-cockroachdb).
```

Comment:
We should probably change this direction slightly to clarify this - in some cases OpenShift shows the image SHA instead of the image tag, and the customer will need to use the SHA.  Again, this is something we didn't know when we wrote up the README